test/__snapshots__/index.test.js.snap

```diff
@@ -658,11 +705,105 @@ exports[`fixtures build class-properties with microbundle 5`] = `
 "
 `;
 
+exports[`fixtures build custom-babelrc with microbundle 1`] = `
+"Used script: microbundle
+
+Directory tree:
+
+custom-babelrc
+  dist
+    custom-babelrc.js
+    custom-babelrc.js.map
+    custom-babelrc.mjs
+    custom-babelrc.mjs.map
+    custom-babelrc.umd.js
+    custom-babelrc.umd.js.map
+  package.json
+  src
+    index.js
+
+
+Build \\"customBabelrc\\" to dist:
+55 B: custom-babelrc.js.gz
+42 B: custom-babelrc.js.br
+62 B: custom-babelrc.mjs.gz
+48 B: custom-babelrc.mjs.br
+177 B: custom-babelrc.umd.js.gz
+141 B: custom-babelrc.umd.js.br"
+`;
+
+exports[`fixtures build custom-babelrc with microbundle 2`] = `6`;
+
+exports[`fixtures build custom-babelrc with microbundle 3`] = `
+"exports.MyClass=class{async foo(){}};
+//# sourceMappingURL=custom-babelrc.js.map
+"
+`;
+
+exports[`fixtures build custom-babelrc with microbundle 4`] = `
+"class o{async foo(){}}export{o as MyClass};
+//# sourceMappingURL=custom-babelrc.mjs.map
+"
+`;
+
+exports[`fixtures build custom-babelrc with microbundle 5`] = `
+"!function(e,o){\\"object\\"==typeof exports&&\\"undefined\\"!=typeof module?o(exports):\\"function\\"==typeof define&&define.amd?define([\\"exports\\"],o):o((e=e||self).customBabelrc={})}(this,function(e){e.MyClass=class{async foo(){}}});
+//# sourceMappingURL=custom-babelrc.umd.js.map
+"
+`;
+
 exports[`fixtures build custom-source with microbundle 1`] = `
 "Used script: microbundle
 
 Directory tree:
 
+custom-babelrc
+  dist
+    custom-babelrc.js
+    custom-babelrc.js.map
+    custom-babelrc.mjs
+    custom-babelrc.mjs.map
+    custom-babelrc.umd.js
+    custom-babelrc.umd.js.map
+  package.json
+  src
+    index.js
+
+
+Build \\"customBabelrc\\" to dist:
+5.6 kB: custom-babelrc.js.gz
+5.05 kB: custom-babelrc.js.br
+5.6 kB: custom-babelrc.mjs.gz
+5.06 kB: custom-babelrc.mjs.br
+5.67 kB: custom-babelrc.umd.js.gz
+5.12 kB: custom-babelrc.umd.js.br"
+`;
+
+exports[`fixtures build custom-source with microbundle 2`] = `6`;
+
+exports[`fixtures build custom-source with microbundle 3`] = `
```

Comment:
there is a bug in core-js@3 where es.promise get transpiled with `{esmodules:true}` which I'll try to fix upstream. When using core-js@2 it's much cleaner. It's not a microbundle issue